samples/apps/logging/logging.cpp

```diff
@@ -581,6 +582,140 @@ namespace loggingapp
         .set_forwarding_required(ccf::ForwardingRequired::Never)
         .install();
 
+      auto get_historical_range = [&, this](ccf::EndpointContext& args) {
+        // Parse request body
+        const auto query_j =
+          ccf::jsonhandler::get_params_from_query(args.rpc_ctx);
+        const auto in = query_j.get<LoggingGetHistoricalRange::In>();
+
+        // Range must be in order
+        if (in.to_seqno < in.from_seqno)
+        {
+          args.rpc_ctx->set_response_status(HTTP_STATUS_INTERNAL_SERVER_ERROR);
```

Comment:
All of those should use `args.rpc_ctx->set_error()`.

Reply:
Thanks - I've replaced all of the `set_response_status(ERROR)` blocks in `logging.cpp` with `set_error(ERROR)`